### ISSUE TITLE
Investigate possibility of using Web Sockets with SNS and Api Gateway

### ISSUE THREAD

**Issue by nikhilaravi** [dwyl/learn-aws-lambda#22]
A quick google search seems to suggest that this is not possible at the moment 

https://forums.aws.amazon.com/thread.jspa?threadID=205761

> Re: Possible to use API Gateway for Websocket-based connections?
> The lambda function has to stay active as long as the client is connected, which you don't want as you pay per 100 ms.

**Comment by nelsonic** [comment]
Yeah, makes sense. need to figure out how to stream data back to client in the event of a long-running request e.g. to a 3rd party API...
